ext/drsyms/libelftc/rebuild_elftc.sh

```diff
@@ -93,13 +98,13 @@ function build_for_bits() {
   if [[ "${make_cmd}" -eq "pmake" ]]; then
       # XXX: pmake does not seem to clean properly, but bmake is working fine.
       rm -f */*.so
-      svn status | grep '?' | awk '{print $2}' | xargs rm -f
+      # svn status | grep '?' | awk '{print $2}' | xargs rm -f
```

Comment:
Never check in commented-out code -- especially with no surrounding comment explaining *why*.  
 See https://github.com/DynamoRIO/dynamorio/wiki/Code-Style#commenting-conventions number 5

Reply:
This code erroneously work in all cases (not only with "pmake". And with these patch these sequence of commands will fail (because we created new directory, doesn't existed in oiginal project)

Reply:
If code should be changed, then change it or have TODO comments about fixing it: silently commenting it out without any explanation is not considered clean coding and makes the code painful to read and maintain.